front/src/applications/editor/tools/pointEdition/__tests__/utils.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { formatSignalingSystems } from 'applications/editor/tools/pointEdition/utils';
+
+import type { SignalEntity } from '../types';
+
+describe('formatSignalingSystems', () => {
+  it('should format BAL signaling systems without default_parameters', () => {
+    const entity: SignalEntity = {
+      type: 'Feature',
+      objType: 'Signal',
+      properties: {
+        id: '1',
+        extensions: {
+          sncf: {},
+        },
+        logical_signals: [
+          {
+            signaling_system: 'BAL',
+            next_signaling_systems: [],
+            settings: { Nf: 'true' },
+            conditional_parameters: [],
+          },
+        ],
+      },
+      geometry: {
+        geometries: [],
+        type: 'GeometryCollection',
+      },
+    };
+    expect(formatSignalingSystems(entity)).toEqual([
```

Comment:
Modify the function and then give it only 1 logicalSignal
```suggestion
    expect(formatSignalingSystem(logicalSignal)).toEqual([
```